### ISSUE TITLE
作者您好，请问为什么我跑traffic的实验720预测720的时候前面的结果都能正常运行，就是在获得结果的时候会killed，batchsize已经调成很小了。显卡使用的是3090ti

### ISSUE THREAD

**Issue by klayc-gzl** [lss-1138/SparseTSF#11]
![image](https://github.com/user-attachments/assets/85e4a554-a461-441f-977b-3d979d5d14a9)

**Comment by lss-1138** [comment]
您好，可能是内存不足的原因（不是显存）。

测试阶段会把真实值、预测值等存起来。由于Traffic这个数据集比较大，可能会内存不足。

可以确认下你的内存是否足够。另外，可以试下把这关于inputx 的代码注释掉（inputx 不影响到测试阶段指标的计算）：

inputx.append(batch_x.detach().cpu().numpy())
inputx = np.concatenate(inputx, axis=0)
inputx = inputx.reshape(-1, inputx.shape[-2], inputx.shape[-1])

**Comment by klayc-gzl** [comment]
好的，谢谢您，注释掉inputx的相关内容后，能成功了！
